terraform/aws/modules/efs/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "creation_token" {
+  type    = string
+  default = "test-efs"
+
+}
+variable "performance_mode" {
+  type    = string
+  default = "generalPurpose"
+
+}
+variable "throughput_mode" {
+  type    = string
+  default = "bursting"
+
+}
+variable "encryption_set" {
+  type    = bool
+  default = true
+}
+variable "kms_key_id" {
```

Comment:
@angeleenasunny , I believe this variable is not needed anymore?

Reply:
@qburst-praven This is removed.